### ISSUE TITLE
Question related to Config section

### ISSUE THREAD

**Issue by olsh** [heroku/12factor#189]
Thanks for the doc 👍 

I'have some questions regarding this part:
https://12factor.net/config

> The twelve-factor app stores config in environment variables (often shortened to env vars or env). Env vars are easy to change between deploys without changing any code; unlike config files, there is little chance of them being checked into the code repo accidentally; and unlike custom config files, or other config mechanisms such as Java System Properties, they are a language- and OS-agnostic standard.

Are you talking about [System Environment Variables](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682653%28v=vs.85%29.aspx)?

If yes, there is a question:

If a new developer clones a repository he needs to know and set all the environment variables to run the app? 
Or if somebody adds a new configuration parameter, other developers should be notified about the change and add the parameter to their environment variables? 

Would it be better to store the template config file in the repo and ignore actual config file? 
With this approach, developers only need to copy the template and modify the config as they want plus the get default parameters.
On deploy, the deployment software can use the template and replace variables in the file.

**Comment by sblancov** [comment]
Yes to the first question. "OS-agnostic" is written so it will work in windows/unix.

Yes to the second question too.

You propose a template and ignore it. I have tried this and it has no advantage. You, anyway, have to ask for values and if values change, then other developers should be notified too.

Maybe a configuration service is the solution.
